### DISCUSSION TITLE
Search on catalyst Channel not returning anything

### DISCUSSION THREAD

**Discussion by Durph09** [bigcommerce/catalyst#1082] [Q&A] [answered]
Currently have a sandbox store, and began working on a headless site with Catalyst.  If I use the default channel (1) with the store search works, but I use the catalyst channel the results are empty.  I've hard coded values into filters and still no results.   I can get results if I go through category or products, not through the search endpoint.  Any ideas would be greatly appreciated.  

query getProductSearchResults(
      $first: Int
      $last: Int
      $after: String
      $before: String
      $filters: SearchProductsFiltersInput!
      $sort: SearchProductsSortInput
    ) {
      site {
        
        search {
          searchProducts(filters: $filters, sort: $sort) {
            products(first: $first, after: $after, last: $last, before: $before) {
              pageInfo {
                ...PageDetails
              }
              collectionInfo {
                totalItems
              }
              edges {
                node {
                  ...ProductCardFragment
                }
              }
            }
            filters {
              edges {
                node {
                  __typename
                  name
                  isCollapsedByDefault
                  ... on BrandSearchFilter {
                    displayProductCount
                    brands {
                      pageInfo {
                        ...PageDetails
                      }
                      edges {
                        cursor
                        node {
                          entityId
                          name
                          isSelected
                          productCount
                        }
                      }
                    }
                  }
                  ... on CategorySearchFilter {
                    displayProductCount
                    categories {
                      pageInfo {
                        ...PageDetails
                      }
                      edges {
                        cursor
                        node {
                          entityId
                          name
                          isSelected
                          productCount
                          subCategories {
                            pageInfo {
                              ...PageDetails
                            }
                            edges {
                              cursor
                              node {
                                entityId
                                name
                                isSelected
                                productCount
                              }
                            }
                          }
                        }
                      }
                    }
                  }
                  ... on ProductAttributeSearchFilter {
                    displayProductCount
                    filterName
                    attributes {
                      pageInfo {
                        ...PageDetails
                      }
                      edges {
                        cursor
                        node {
                          value
                          isSelected
                          productCount
                        }
                      }
                    }
                  }
                  ... on RatingSearchFilter {
                    ratings {
                      pageInfo {
                        ...PageDetails
                      }
                      edges {
                        cursor
                        node {
                          value
                          isSelected
                          productCount
                        }
                      }
                    }
                  }
                  ... on PriceSearchFilter {
                    selected {
                      minPrice
                      maxPrice
                    }
                  }
                  ... on OtherSearchFilter {
                    displayProductCount
                    freeShipping {
                      isSelected
                      productCount
                    }
                    isFeatured {
                      isSelected
                      productCount
                    }
                    isInStock {
                      isSelected
                      productCount
                    }
                  }
                }
              }
            }
          }
        }
      }
    }

**Comment by bookernath** [comment, marked as answer]
Hi @Durph09, the most likely problem is that multi-storefront is not enabled on your store. Our support team can flip this on for you.

**Reply by Durph09** [reply]
Thank you @bookernath, it was a set up issue that support straightened out.  Just want to say really nice job on Catalyst! the more I dig in the more impressed I am.   We'll be migrating.
